atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -215,8 +238,17 @@ public static LockAndTimestampServices createLockAndTimestampServices(
             Environment env,
             Supplier<RemoteLockService> lock,
             Supplier<TimestampService> time) {
+        return createLockAndTimestampServices(config, env, lock, time, UserAgents.DEFAULT_USER_AGENT);
+    }
+
+    public static LockAndTimestampServices createLockAndTimestampServices(
```

Comment:
We could, though I think the only case where this is used without going through a transaction manager is in Dagger (for the CLIs), so I'm not sure it's worth expanding the public API for this. Feel free to push back.